internal/util/bundleutil/bundleutil.go

```diff
@@ -0,0 +1,239 @@
+			return err
+		}
+	}
+	log.Infof("Bundle metadata generated suceessfully")
+	return nil
+}
+
+// WriteScorecardConfig adds the default scorecard config containing OLM tests.
+func (meta *BundleMetaData) WriteScorecardConfig() error {
+	scorecardDir := filepath.Join(meta.BundleDir, "tests/scorecard")
+
+	// Create directory for scorecard config
+	if err := os.MkdirAll(scorecardDir, projutil.DirMode); err != nil {
+		return err
+	}
+
+	log.Info(fmt.Sprintf("writing scorecard config in %s", scorecardDir))
+
+	return writeScorecardConfig(filepath.Join(scorecardDir, "config.yaml"))
+}
+
+func writeScorecardConfig(path string) error {
+	f, err := os.OpenFile(path, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0666)
+	if err != nil {
+		return err
+	}
+
+	defer func() {
+		if err := f.Close(); err != nil {
+			log.Error(err)
+		}
+	}()
+	err = ioutil.WriteFile(path, []byte(scorecardTemplate), projutil.FileMode)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// CopyOperatorManifests copies packagemanifestsDir/manifests to bundleDir/manifests
+func (meta *BundleMetaData) CopyOperatorManifests() error {
+	return copyOperatorManifests(meta.PkgmanifestPath, filepath.Join(meta.BundleDir, defaultManifestDir))
+}
+
+func copyOperatorManifests(src, dest string) error {
+
+	srcInfo, err := os.Stat(src)
+	if err != nil {
+		return fmt.Errorf("error reading source directory %v", err)
+	}
+
+	if err := os.MkdirAll(dest, srcInfo.Mode()); err != nil {
+		return err
+	}
+
+	srcFiles, err := ioutil.ReadDir(src)
+	if err != nil {
+		return err
+	}
+
+	for _, f := range srcFiles {
+		srcPath := filepath.Join(src, f.Name())
+		destPath := filepath.Join(dest, f.Name())
+
+		if f.IsDir() {
+			// TODO(verify): we may have to log an error here instead of recursively copying
+			// if there are no subfolders allowed under manifests dir of a packagemanifest.
+			if err = copyOperatorManifests(srcPath, destPath); err != nil {
+				return err
+			}
+		} else {
+			srcFile, err := os.Open(srcPath)
+			if err != nil {
+				return err
+			}
+			defer srcFile.Close()
+
+			destFile, err := os.Create(destPath)
+			if err != nil {
+				return err
+			}
+			defer destFile.Close()
+
+			_, err = io.Copy(destFile, srcFile)
+			if err != nil {
+				return err
+			}
+		}
+	}
+	return nil
+}
+
+// BuildBundleImage builds the bundle image with the provided command or using
+// docker build command.
+func (meta *BundleMetaData) BuildBundleImage(tag string) error {
+
+	img := fmt.Sprintf("%s:%s", meta.BaseImage, tag)
+
+	if len(meta.BuildCommand) != 0 {
+		// TODO(varsha): Make this more user friendly by accepting a template which
+		// can executed in each bundle subdirectory.
+		log.Infof("Using the specified command to build image %s", img)
+		cmd := exec.Command(meta.BuildCommand, img)
+		if err := cmd.Run(); err != nil {
+			return err
+		}
+	} else {
+		output, err := exec.Command("docker", "build", "-f", filepath.Join(meta.BundleDir, "bundle.Dockerfile"), "-t", img, ".").Output()
```

Comment:
Use [`CombinedOutput()`](https://golang.org/pkg/os/exec/#Cmd.CombinedOutput) to combine both stdout and stderr in output.
```suggestion
		output, err := exec.Command("docker", "build", "-f", filepath.Join(meta.BundleDir, "bundle.Dockerfile"), "-t", img, ".").CombinedOutput()
```